lib/cli.js

```diff
@@ -120,3 +130,13 @@ npmCheck(options)
         }
         process.exit(1);
     });
+
+var SUPPORTED_INSTALLERS = ['npm', 'pnpm', 'ied'];
```

Comment:
The readme mentions ied, so I added it as well. Although, currently `preferred-pm` cannot detect it. I am not sure it ever will. Ied seems to be not maintained anymore

Reply:
Yarn can be added but it will require additional work. Yarn's CLI is not compatible with npm's